lib/react.dart

```diff
@@ -1155,19 +1110,8 @@ class SyntheticFormEvent extends SyntheticEvent {
     dynamic target,
     num timeStamp,
     String type,
-  ) : super(
-            bubbles,
-            cancelable,
-            currentTarget,
-            defaultPrevented,
-            preventDefault,
-            stopPropagation,
-            eventPhase,
-            isTrusted,
-            nativeEvent,
-            target,
-            timeStamp,
-            type) {}
+  ) : super(bubbles, cancelable, currentTarget, defaultPrevented, preventDefault, stopPropagation, eventPhase,
+            isTrusted, nativeEvent, target, timeStamp, type) {}
```

Comment:
"Oh noooo is there anything we can do here?"

Reply:
add a trailing comma probably